spec/support/marmotta_clear.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.configure do |config|
+  config.before(:each) do
```

Comment:
Really?  Before *each*?  That seems like a potential bottleneck for tests that have no marmotta concerns.